<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomaatioMuokkain</class>
 <widget class="QWidget" name="AutomaatioMuokkain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="opastinNappi">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="ratapiha-kauko.qrc">
         <normaloff>:/r/pic/po-aja.png</normaloff>
         <normalon>:/r/pic/automaatioopastin.png</normalon>:/r/pic/po-aja.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="opastinLabel">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Opastin</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="automaatioLista"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="herateLabel">
         <property name="text">
          <string>Heräte</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="herateNappi">
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="maaliLabel">
         <property name="text">
          <string>Maali</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="maaliNappi">
           <property name="text">
            <string>?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="atraideNappi">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="ratapiha-kauko.qrc">
             <normaloff>:/r/pic/reitit.png</normaloff>:/r/pic/reitit.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="ehtoLabel">
         <property name="text">
          <string>Ehto</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ehtoEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="priorLabel">
         <property name="text">
          <string>Prioriteetti</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="prioriteettiSpin">
         <property name="minimum">
          <number>-100</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="viiveLabel">
         <property name="text">
          <string>Viive</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="viiveSpin">
         <property name="suffix">
          <string> s</string>
         </property>
         <property name="maximum">
          <number>6000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="aikatauluViiveCheck">
         <property name="text">
          <string>Aikataulun mukaan</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="tyyppiLabel">
         <property name="text">
          <string>Kulkutie</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="tyyppiCombo">
         <item>
          <property name="text">
           <string>Junakulkutie</string>
          </property>
          <property name="icon">
           <iconset resource="ratapiha-kauko.qrc">
            <normaloff>:/r/pic/vihrea.png</normaloff>:/r/pic/vihrea.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Vaihtokulkutie</string>
          </property>
          <property name="icon">
           <iconset resource="ratapiha-kauko.qrc">
            <normaloff>:/r/pic/ro-aja.png</normaloff>:/r/pic/ro-aja.png</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Varatun raiteen junakulkutie</string>
          </property>
          <property name="icon">
           <iconset resource="ratapiha-kauko.qrc">
            <normaloff>:/r/pic/kulkutieVaratulle.png</normaloff>:/r/pic/kulkutieVaratulle.png</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="kerranCheck">
         <property name="text">
          <string>Yritä vain kerran</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="lisaaNappi">
         <property name="icon">
          <iconset resource="ratapiha-kauko.qrc">
           <normaloff>:/r/pic/lisaa.png</normaloff>:/r/pic/lisaa.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="poistaNappi">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="ratapiha-kauko.qrc">
           <normaloff>:/r/pic/poista.png</normaloff>:/r/pic/poista.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="valmisNappi">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="ratapiha-kauko.qrc">
           <normaloff>:/r/pic/hyvaksy.png</normaloff>:/r/pic/hyvaksy.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="heratteenLista"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ratapiha-kauko.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>aikatauluViiveCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>viiveSpin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
